src/lavinmq/rough_time.cr

```diff
@@ -1,26 +1,43 @@
 module RoughTime
-  @@utc = Time.utc
-  @@unix_ms : Int64 = @@utc.to_unix_ms // 100 * 100
-  @@monotonic = Time.monotonic
+  {% if flag?(:realtime) %}
+    @[AlwaysInline]
```

Comment:
I'm pretty sure the compiler will figure that out without the attribute

Reply:
Yes, most likely.